components/viaduct/src/backend/ffi.rs

```diff
@@ -64,11 +64,6 @@ impl Backend for FfiBackend {
         };
 
         if let Some(exn) = response.exception_message {
-            log::error!(
-                // Well, we caught *something* java wanted to tell us about, anyway.
-                "Caught network error (presumably). Message: {:?}",
-                exn
-            );
```

Comment:
Well because of the protobuf bug I uncovered this line was never called, but it will.